atd-vze/src/views/Crashes/crashGridTableParameters.js

```diff
@@ -58,21 +58,78 @@ export const crashGridTableColumns = {
     label_table: "Collision Description",
     type: "String",
   },
-  "units { unit_desc_lkp { label } }": {
-    searchable: false,
-    sortable: false,
-    label_table: "Unit Description",
-    type: "String",
-    hidden: true,
-  },
-  is_manual_geocode: {
-    searchable: false,
-    sortable: true,
-    label_table: "Manual geocode",
-    type: "Boolean",
-  },
```

Comment:
this really slows down the main crashes list and we only need it for the location details crash list